### ISSUE TITLE
Changing Fonts inside nuxt.config removes css folder

### ISSUE THREAD

**Issue by estnml** [nuxt-modules/google-fonts#169]
Hi, this my nuxt.config.ts

```typescript
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
    devtools: { enabled: true },
    modules: [
        "@nuxtjs/tailwindcss",
        "@nuxtjs/google-fonts",
        "@vueuse/nuxt"
    ],
    components: [
        { path: "~/components", pathPrefix: false }
    ],
    googleFonts: {
        inject: true,
        base64: false,
        download: true,
        outputDir: "assets",
        stylePath: "css/external/google-fonts.css",
        fontsDir: "fonts",
        families: {
            Oxygen: true
        }
    },
    css: ["~/assets/css/main.css", "~/assets/css/external/google-fonts.css"]
})
``` 
when i replace Oxygen with another font, it deletes the whole css folder and the contents and inserts the google-fonts.css file. So if i have some css files inside the folder and change the desired font, all my css would be gone

**Comment by gibbsy** [comment]
I've also experienced this issue

**Comment by ricardogobbosouza** [comment]
Really...this will solve it https://github.com/datalogix/google-fonts-helper/pull/65

**Comment by ricardogobbosouza** [comment]
Released [v3.2.0](https://github.com/nuxt-modules/google-fonts/releases/tag/v3.2.0)
